selinux: Enable mode change

dperpeet opened the pull request:
With an on/off toggle, the user can now switch between enforcing and
permissive mode, if SELinux is enabled on the system.

andreasn commented:
The line break makes the label and on/off widget feel disconnected.

![screenshot from 2016-05-25 18-04-48](https://cloud.githubusercontent.com/assets/209895/15547230/3282bf92-22a3-11e6-810b-bb77187c6df0.png)

What about something like this?
![screenshot from 2016-05-25 18-10-31](https://cloud.githubusercontent.com/assets/209895/15547418/05371b40-22a4-11e6-886f-96a1dd6c4d27.png)

andreasn commented:
I can cook up some css for these fixes

dperpeet commented:
I added some css to this effect. Also I fixed the on/off to not change via jquery js anymore, only within react.
This is what happens when a user without the relevant permissions tries to toggle the mode button:
![screenshot from 2016-05-25 23-04-10](https://cloud.githubusercontent.com/assets/8711649/15556209/120741c8-22cd-11e6-8afa-850620e878f7.png)

dperpeet commented:
Adding the tooltip is non-trivial in react, so I'm leaving that to a follow-up.

dperpeet commented:
this should go into the next release

larsu commented on pkg/selinux/selinux-client.js:
```diff
@@ -0,0 +1,113 @@
+// how often to check the status [milliseconds]
+var pollingInterval = 10000;
+
+var statusCommand = "sestatus";
```
Ow wow, that's the only way to find out about the status? It would be good to be notified about changes so that the UI doesn't lag up to 10 seconds after running `setenforce`.

dperpeet replied:
> Ow wow, that's the only way to find out about the status? It would be good to be notified about changes so that the UI doesn't lag up to 10 seconds after running setenforce.

Yeah. There is no good alternative. Everything else has to be configured. In the near future, I'm hoping for a dbus api.

larsu commented:
@andreasn there's a lot of space to include explanatory text below "Enforced Security Mode". Do you think it makes sense to add that there?

Looks good otherwise.

dperpeet commented:
> @andreasn there's a lot of space to include explanatory text below "Enforced Security Mode". Do you think it makes sense to add that there?

I think an explanation there would go against our pattern of not cluttering the ui with such explanations. A follow-up will contain the popup tooltip, but that is non-trivial in react from what I've seen (unless we use bootstrap-react, which seems heavy-weight to pull in and that discussion seems out of scope here).
